react/components/AddToList/Footer.tsx

```diff
@@ -1,32 +1,25 @@
-import React, { Component, ReactNode } from "react"
-import { FormattedMessage, InjectedIntlProps, injectIntl, IntlShape } from 'react-intl'
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
 import { Button } from 'vtex.styleguide'
+
 import wishlist from '../../wishList.css'
 
 interface FooterProps {
-  intl: IntlShape
   isLoading?: boolean
   changedLists: any[]
   onClick: () => void
 }
 
-class Footer extends Component<FooterProps & InjectedIntlProps, {}> {
-  public render(): ReactNode {
-    const { intl, isLoading, changedLists, onClick } = this.props
-    return (
-      <div className={wishlist.applyButton}>
-        <Button
-          vatiation="primary"
-          disabled={!changedLists.length}
-          block
-          onClick={onClick}
-          isLoading={isLoading}
-        >
-          <FormattedMessage id="wishlist-apply" />
-        </Button>
-      </div>
-    )
-  }
-}
-
-export default injectIntl(Footer)
\ No newline at end of file
+export default ({ isLoading, changedLists, onClick }: FooterProps): JSX.Element => (
```

Comment:
create a variable for the component, like `const Footer = ...`, so the devtools can automatically get the component displayName